packages/vkui/src/components/Mark/Mark.e2e.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import { test } from '@vkui-e2e/test';
+import { MarkPlayground } from './Mark.e2e-playground';
+
+test('Mark', async ({ mount, expectScreenshotClippedToContent, componentPlaygroundProps }) => {
```

Comment:
```suggestion
test(Mark, async ({ mount, expectScreenshotClippedToContent, componentPlaygroundProps }) => {
```

`toString` вернёт название функции

при таком обозначении компонента в тесте, после его переименования, IDE также и тут изменит

пару раз встречал кейсы, когда компонент переименовали, а в названии теста забыли

Reply:
фикс

Reply:
<img width="967" alt="image" src="https://github.com/VKCOM/VKUI/assets/14944123/26e46da8-7dd9-4f3a-82fc-1ca498323739">

![image](https://github.com/VKCOM/VKUI/assets/14944123/577c83a1-30de-402c-b8c7-ca6435962b5e)

`toString` не сработал, заюзал `name`

Reply:
Эх, хорошая была